source/slang/slang-compiler.cpp

```diff
@@ -561,6 +561,21 @@ namespace Slang
         return translationUnit;
     }
 
+    static TranslationUnitRequest* _getTranslationUnit(
```

Comment:
The name of this function is a bit deceptive, since one might expect `_getTranslationUnit(req, 0)` to get the first translation unit in order, while in actuality it gets the translation unit associated with the first entry point. Maybe `_getTranslationUnitForEntryPoint`?